Gemfile

```
@@ -15,4 +15,4 @@ gem 'html-proofer'
 gem 'html-proofer-unrendered-markdown'
 gem 'jekyll-redirect-from'
 gem "kramdown", ">= 2.3.0"
-gem 'swedbank-pay-design-guide-jekyll-theme', '~> 1.7'
+gem 'swedbank-pay-design-guide-jekyll-theme', '~> 1.8'
```

1.8.1 brings in the `menu_order` renaming.

```suggestion
gem 'swedbank-pay-design-guide-jekyll-theme', '~> 1.8.1'
```